### ISSUE TITLE
When playing where Bene Gesserit advisors can share strongholds with allies, AI acts as though they cannot

### ISSUE THREAD

**Issue by sirrus233** [ronaldossendrijver/treachery.online#15]
This is hard to prove without looking at the AI code, but I've observed it enough times now that I'm pretty sure it's real.

By the normal Advanced rules, Bene Gesserit advisors are not allowed to share a stronghold with their ally. But treachery.online allows us to override this rule. When playing such a game, the AI will *always* prioritize moving out of a shared stronghold once the alliance is made. This seems to happen whether the AI is playing Bene Gesserit, or the Bene Gesserit's ally.

Furthermore, if Bene Gesserit has an advisor in a stronghold, an AI ally will refuse to ship troops into that stronghold, even if doing so would result in winning the game.

**Comment by ronaldossendrijver** [comment]
Hi! Advisors can't be in the same territories as their allies according to the standard GaleForceNine rules, unless you enable that as a houserule.

**Comment by sirrus233** [comment]
Yes, understood. But treachery.online supports that house rule. So when that house rule is enabled, might we not want the AI play appropriately?

**Comment by ronaldossendrijver** [comment]
Right, I'm sorry, I didn't read your description well enough. I understand now that the bot does not correctly take into account the advisor-doesn't conflicy with ally houserule. Will be improved in version 165!
